src/components/ProjectHeader.jsx

```diff
@@ -3,48 +3,79 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router';
 import { config } from '../config';
 
-const ProjectHeader = ({ showTitle }) =>
-  <div className="project-header">
-    {showTitle && (
-      <h1 className="main-title">Anti-Slavery Manuscripts</h1>
-    )}
-    <nav className="project-header__nav">
-      <Link
-        activeClassName="project-header__link--active"
-        className="project-header__link"
-        onlyActiveOnIndex
-        to="/"
-      >
-        Home
-      </Link>
-      <Link
-        activeClassName="project-header__link--active"
-        className="project-header__link"
-        to="/classify"
-      >
-        Transcribe
-      </Link>
-      <Link
-        activeClassName="project-header__link--active"
-        className="project-header__link"
-        to="/about"
-      >
-        About
-      </Link>
-      <a
-        className="project-header__link"
-        href={config.zooniverseLinks.host + 'projects/' + config.zooniverseLinks.projectSlug + '/collections'}
-      >
-        Collect
-      </a>
-      <a
-        className="project-header__link"
-        href={config.zooniverseLinks.host + 'projects/' + config.zooniverseLinks.projectSlug + '/talk'}
-      >
-        Talk
-      </a>
-    </nav>
-  </div>;
+class ProjectHeader extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.aboutClick = this.aboutClick.bind(this);
+    this.talkClick = this.talkClick.bind(this);
+  }
+
+  aboutClick() {
+    if (this.context.googleLogger) {
```

Comment:
Wait, `React.context` ? Hissss! Unclean, unclean! Legend tells that React.context was forged by Sauron himself in the heart of Orodruin to tempt the hearts of developers. _Its power comes at great cost._

Theatrics aside, I can see where using context _could_ make sense in our case. We generally avoid React.context because it might be patched out in future versions of React, but if we can accept the ASM project's limited lifespan, we can accept the risks of using context.